src/vivarium_csu_alzheimers/constants/data_keys.py

```diff
@@ -31,19 +32,22 @@ def log_name(self):
 
 
 class __Alzheimers(NamedTuple):
-    # Prevalence scale factor will scale the population and "fertility" of the model
-    PREVALENCE_SCALE_FACTOR: str = (
-        "cause.alzheimers_disease_and_other_dementias.prevalence_scale_factor"
-    )
-    # Prevalence will just be set to 1 so all simulants are created with the disease
     PREVALENCE: str = "cause.alzheimers_disease_and_other_dementias.prevalence"
+    BBBM_CONDITIONAL_PREVALANCE: str = "cause.bbbm.prevalence"
```

Comment:
I find it confusing that we have so many version of these strings --- is there a convention we can adopt that will make the all-caps stuff match the double-quoted stuff more precisely?  And should the convention require that the term after "cause" is "alzheimers" if this is all in the `__Alzheimers` class?

Reply:
Same with MCI keys in this class.